.gitignore

```diff
@@ -11,6 +11,7 @@ zombienet-tests/yarn.lock
 zombienet-tests/bin
 lcov.info
 benchmarking_errors.txt
+staging/
```

Comment:
Just a question if you add this here it should not possible to remove it but, the `.gitignore` in `staging` folder that ignore all files instead `.gitignore` should already do the job.